safe-global/web-core
# fix: Add policy pages

## What it solves

Resolves #1722 

## How this PR fixes it

- Adds a new page for Privacy policy, Cookie policy, Terms and Imprint
- Links those pages in the footer
- Displays the footer on those pages
- Hides the sidebar on those pages
- Shows a one time terms popup for users who have previously dismissed the cookie banner (existing users)

## How to test it

1. Open the Safe
2. Observe a Terms banner showing up in the bottom corner
3. Open the Safe in a private tab and observe only the cookie banner showing
4. Accept the cookie banner and observe the Terms banner not showing up
5. Reload the page and observe the Terms banner not showing up
6. Go to Settings to see the Footer
7. Click on any of the three links to see the page
8. Compare to https://www.notion.so/gnosis-safe/Terms-policy-update-f3827a6f724340deae9554d24afe8c29
9. Create or Load a new safe and click on terms of use or privacy policy and observe seeing the new pages

## Screenshots

<img width="502" alt="Screenshot 2023-03-03 at 11 06 23" src="https://user-images.githubusercontent.com/5880855/222692334-b3f264a3-f512-41d5-bda5-85b2bb7de2cd.png">
